src/components/Articles.jsx

```diff
@@ -1,21 +1,53 @@
-import React from "react";
+import axios from "axios";
+import { useState, useEffect } from "react";
 import ArticleBlock from "./ArticleBlock";
 
-function Articles({ articles, isLoading }) {
+function Articles() {
+  const [articles, setArticles] = useState([]);
+  const [articleId, setArticleId] = useState();
+  const [oneArticle, setOneArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    axios
+      .get("https://nc-news-api-qfui.onrender.com/api/articles")
```

Comment:
This axios function should really be in a file called api.js with the other API calls 👍